File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/video_generation_request.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class VideoGenerationRequest:
    """Video generation request"""
    prompt: str
    model: str = None
    seconds: str = None
    size: str = None
    quality: str = None
    width: int = None
    height: int = None
    seed: int = None
    user: str = None
    negative_prompt: str = None
    image_urls: List[str] = None
    aspect_ratio: str = None
    response_format: str = None

File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/VideoGenerationRequest.kt
package com.sdkwork.ai

data class VideoGenerationRequest(
    val prompt: String? = null,
    val model: String? = null,
    val seconds: String? = null,
    val size: String? = null,
    val quality: String? = null,
    val width: Int? = null,
    val height: Int? = null,
    val seed: Int? = null,
    val user: String? = null,
    val negativePrompt: String? = null,
    val imageUrls: List<String>? = null,
    val aspectRatio: String? = null,
    val responseFormat: String? = null
)
